### DISCUSSION TITLE
Nano Matter connects, but xG24 ExpKit not

### DISCUSSION THREAD

**Discussion by metehoca** [SiliconLabs/arduino#58] [Ideas]
Hi, I have Nano Matter and xG24 Explorer Kit. Nano Matter works as expected, but xG24 immediately shows "Can't connect to your device" on Google Home. Used different examples, erasing chip, flashing matter bootloader but no luck. Any ideas?

I'm using Nest Hub 2nd gen.

EDIT: Changed to BLE bootloader and back again to Matter and it worked. IDK why.

**Comment by silabs-bozont** [comment]
Hello @metehoca,
The previous 1.0.0 version used different bootloaders for BLE and Matter - Matter wouldn't work with a BLE bootloader a vice versa.
Since 2.0.0 the bootloaders have been unified, so now all variants use the same one. Also now burning the bootloader erases the device beforehand.
Glad it works!
